src/WindowsAzure/Table/ITableSet.cs

```diff
@@ -165,5 +167,51 @@ public interface ITableSet<TEntity> : IOrderedQueryable<TEntity> where TEntity :
         /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>Result.</returns>
         Task RemoveAsync(IEnumerable<TEntity> entities, CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <summary>
+        ///    Creates the table if it does not already exist.
+        /// </summary>      
+        /// <param name="requestOptions">A <see cref="T:Microsoft.WindowsAzure.Storage.Table.TableRequestOptions"/> object that specifies additional options for the request.</param>
+        /// <param name="operationContext">An <see cref="T:Microsoft.WindowsAzure.Storage.OperationContext"/> object that represents the context for the current operation.</param>
+        /// <returns>
+        /// <c>true</c> if table was created; otherwise, <c>false</c>.
+        /// </returns>                  
+        void CreateIfNotExists(TableRequestOptions requestOptions = null, OperationContext operationContext = null );
+
+        /// <summary>
+        ///     Initiates an asynchronous operation to create a table if it does not already exist.        
+        /// </summary>        
+        /// <returns>
+        /// A <see cref="T:System.Threading.Tasks.Task`1"/> object of type <c>bool</c> that represents the asynchronous operation.
+        /// </returns>        
+        Task<bool> CreateIfNotExistsAsync();
+
+        /// <summary>
+        ///     Initiates an asynchronous operation to create a table if it does not already exist.        
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="T:System.Threading.CancellationToken"/> to observe while waiting for a task to complete.</param>
+        /// <returns>
+        /// A <see cref="T:System.Threading.Tasks.Task`1"/> object of type <c>bool</c> that represents the asynchronous operation.
+        /// </returns>
+        Task<bool> CreateIfNotExistsAsync(CancellationToken cancellationToken);
```

Comment:
Change argument in this method like that: CancellationToken cancellationToken = default (CancellationToken)
And remove all others CreateIfNotExistsAsync method overloads